console/console-init/ui/src/modules/connection/components/ConnectionsToggleGroup/ConnectionsToggleGroup.tsx

```diff
@@ -91,6 +91,8 @@ const ConnectionsToggleGroup: React.FunctionComponent<IConnectionsToggleGroupPro
           {filterSelected && filterSelected.toLowerCase() === "hostname" && (
             <InputGroup>
               <TypeAheadSelect
+                id="connections-toggle-grp-select-hostname-typeahead"
```

Comment:
We should avoid using terms like typeahead, as it's too patternfly specific. We should use generic terms like 'dropdown' or 'auto-populate' which is understandable to all.